docs/concepts/storage/persistent-volumes.md

```diff
@@ -104,6 +108,10 @@ spec:
 
 However, the particular path specified in the custom recycler pod template in the `volumes` part is replaced with the particular path of the volume that is being recycled.
 
+#### Deleting
+
+For volume plugns that support the Delete reclaim policy, deletion removes both the `PersistentVolume` object from Kubernetes, as well as deleting the associated storage asset in external infrastructure (such as an AWS EBS, GCE PD, Azure Disk, or Cinder volume). Volumes that were dynamically provisioned are always deleted. If that is not desired, the only current option is to edit or patch the PV after it is created. See [Change the Reclaim Policy of a PersistentVolume](https://kubernetes.io/docs/tasks/administer-cluster/change-pv-reclaim-policy/)
```

Comment:
* "plugns" -> "plugins"
* add "the": "...storage asset in `the` external infrastructure..."
* don't need parentheses: "...external infrastructure, such as an AWS EBS, GCE PD, Azure Disk, or Cinder volume."
* word order: "If that is not desired, currently, the only option is to edit or patch the PV after it is created."